torch/jit/_script.py

```diff
@@ -841,6 +849,93 @@ def call_prepare_scriptable_func(obj):
     memo: Dict[int, torch.nn.Module] = {}
     return call_prepare_scriptable_func_impl(obj, memo)
 
+def _script_pdt(obj, optimize=None, _frames_up=0, _rcb=None, example_inputs: Optional[List[Tuple]] = None):
+    # This is a private API, intended for internal use only . Usage of this API is only for experimental
+    # purposes only and is highly discouraged to be used.
+    global type_trace_db
```

Comment:
TODO for future:
We should avoid using global variable for `type_trace_db` because this allows concurrent scripting to interfere with each other, but this is not critical for prototyping.

Reply:
Agreed!